packages/vscode-extension/src/extension.ts

```diff
@@ -78,17 +78,17 @@ export async function activate(context: ExtensionContext) {
 
   commands.executeCommand("setContext", "RNIDE.sidePanelIsClosed", false);
 
-  async function showIDEPanel(fileName?: string, lineNumber?: number) {
+  async function showIDEPanel() {
     await commands.executeCommand("setContext", "RNIDE.sidePanelIsClosed", false);
 
     const panelLocation = workspace
       .getConfiguration("RadonIDE")
       .get<PanelLocation>("panelLocation");
 
     if (panelLocation !== "tab") {
-      SidePanelViewProvider.showView(context, fileName, lineNumber);
```

Comment:
Why is context not necessery in Side panel while it remains necessary in tabpanel?

Reply:
`showView` doesn't use it at all, so I removed it 💁‍♂️ 

From what I quickly went through, SidePanelViewProvider gets context in the constructor and, being a static method, fails gracefully when it isn't provided. I didn't go deeper, as I assume this is outside the scope of this PR.

Reply:
Fair enough